google/cloud/storage/oauth2/service_account_credentials_test.cc

```diff
@@ -352,6 +359,102 @@ TEST_F(ServiceAccountCredentialsTest, ParseMissingFieldFails) {
   }
 }
 
+/// @test Verify that refreshing a credential updates the timestamps.
+TEST_F(ServiceAccountCredentialsTest, RefreshingUpdatesTimestamps) {
+  auto info = ParseServiceAccountCredentials(kJsonKeyfileContents, "test");
+  ASSERT_STATUS_OK(info);
+
+  auto make_request_assertion = [&info](long timestamp) {
+    return [timestamp, &info](std::string const& p) {
+      std::string const prefix =
+          std::string("grant_type=") + kGrantParamEscaped;
+      EXPECT_THAT(p, HasSubstr(prefix));
+      EXPECT_THAT(p, HasSubstr("&assertion="));
```

Comment:
And similarly for the next part beginning with "&assertion=".

Reply:
See above.